Bottom: e18ea3610ebc1976db34a9e89f057691ed4deafc
Top:    00ecb33c93e6fbbc084cf1873f2ad7d1e670cdff
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-14 16:15:33 -0700

Refresh of SBStream

---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 94b16f7f88c..5438cdba890 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -901,6 +901,13 @@ class FileHandleTestCase(lldbtest.TestBase):
         with open(self.out_filename, 'r') as f:
             self.assertEqual(f.read().strip(), "zork")
 
+        with open(self.out_filename, 'w') as f:
+            stream = lldb.SBStream()
+            stream.RedirectToFileHandle(f, True)
+            stream.Print("Yendor")
+        with open(self.out_filename, 'r') as f:
+            self.assertEqual(f.read().strip(), "Yendor")
+
         stream = lldb.SBStream()
         f = open(self.out_filename,  'w')
         stream.RedirectToFile(lldb.SBFile.Create(f, borrow=False))
